app/forms/DeclarationPage.scala

```diff
@@ -16,4 +16,18 @@
 
 package forms
 
-trait DeclarationPage
+import models.DeclarationType.{CLEARANCE, DeclarationType}
+import models.viewmodels.TariffContentKey
+
+trait DeclarationPage {
+  def defineTariffContentKeys(decType: DeclarationType): Seq[TariffContentKey] = Seq.empty
+
+  def defineJourneySpecificKey(decType: DeclarationType): String = "common"
+}
+
+abstract class ClearanceModifierPage extends DeclarationPage {
```

Comment:
Why an abstract class? Couldn't it still be a trait?

Reply:
:+1: